index.html

```diff
@@ -390,8 +390,8 @@ <h3>MathML Element Mappings</h3>
 		    <span class="objattr">Object Attribute: <code>tag:ms</code></span>
 		  </td>
 		  <td class="axapi">
-		    <span class="role">AXRole: <code>TBD</code></span><br />
-		    <span class="subrole">AXSubrole: <code>TBD</code></span>
+		    <span class="role">AXRole: <code>NSAccessibilityGroupRole</code></span><br />
+		    <span class="subrole">AXSubrole: <code></code></span>
```

Comment:
Mind doing "<nil>" when the AXSubrole value is empty? (Not saying that is an excellent thing to do, but it was done for Core-AAM before my time. And... Consistency. :slightly_smiling_face:

Reply:
yep, done

Reply:
Thanks! Please merge.